dnf-behave-tests/common/output.py

```diff
@@ -36,6 +36,20 @@ def strip_reposync_dnf5(found_lines, line_number):
     if line_number < len(found_lines) and found_lines[line_number].strip() == "Repositories loaded.":
         found_lines.pop(line_number)
 
+def strip_help_dnf4(found_lines, line_number):
+    pass
+
+def strip_help_dnf5(found_lines, line_number):
+    end_of_help_dnf5 = "--disableplugin=PLUGIN_NAME,...        Alias for '--disable-plugin'"
```

Comment:
I understand your intention and I don't know what to do about this exactly, but this is too specific and will break too often...